readthedocs/oauth/services/github.py

```diff
@@ -15,19 +15,20 @@
 
 from ..constants import GITHUB
 from ..models import RemoteOrganization, RemoteRepository
-from .base import Service, SyncServiceError
+from .base import SyncServiceError, UserService
 
 log = structlog.get_logger(__name__)
 
 
-class GitHubService(Service):
+class GitHubService(UserService):
 
     """Provider service for GitHub."""
 
     adapter = GitHubOAuth2Adapter
     # TODO replace this with a less naive check
     url_pattern = re.compile(r"github\.com")
     vcs_provider_slug = GITHUB
+    provider_name = "GitHub"
```

Comment:
Found another way to get the provider name by relying on allauth's provider class without making it an instance method, but will do that in another PR after this one is merged, as it relies on another change.

Reply:
https://github.com/readthedocs/readthedocs.org/pull/11942/commits/54e7ec6dce19a3f7cd0a94cdc0e0004118217098